dask_expr/io/parquet.py

```diff
@@ -1001,17 +1001,16 @@ def fragments_unsorted(self):
 
     @property
     def _fusion_compression_factor(self):
-        if self.operand("columns") is None:
-            return 1
         approx_stats = self.approx_statistics()
         total_uncompressed = 0
         after_projection = 0
-        col_op = self.operand("columns")
+        col_op = self.operand("columns") or self.columns
         for col in approx_stats["columns"]:
             total_uncompressed += col["total_uncompressed_size"]
             if col["path_in_schema"] in col_op:
                 after_projection += col["total_uncompressed_size"]
 
+        total_uncompressed = max(total_uncompressed, 75_000_000)
```

Comment:
I suggest to expose this as a config value

Reply:
https://github.com/dask/dask/pull/11052

Reply:
I'll write docs for this tomorrow or later today after I finished the blog (we need release notes anyway for the trivial shuffles)